packages/fiori/src/ShellBar.ts

```diff
@@ -1293,3 +1358,12 @@ class ShellBar extends UI5Element {
 ShellBar.define();
 
 export default ShellBar;
+
+export type {
+	ShellbarNotificationsClickEventDetail,
```

Comment:
It's more correct to follow the name of the component so rename `ShellbarProfileC...` to `ShellBarProfileC...`